src/renderer/main.tsx

```diff
@@ -143,12 +146,21 @@ export const Main = memo(({ port }: MainProps) => {
                 return prev;
             });
         }
+    }, [response, data, loading, error, backendReady, sendAlert, t]);
 
+    useIpcRendererListener('backend-started', () => {
+        // We know the backend is good at this point, so we can set the retry count to a high number
+        setRetryCount(9999);
+    });
```

Comment:
What exactly is this?

Reply:
hmmm i had originally put this here to avoid it erroring when /nodes wasnt available, but I don't think its needed anymore, since /nodes should be available at the same time the backend starts